src/components/Admin/AdminListings/AdminListingsTable/AdminListingTableRow.tsx

```diff
@@ -36,7 +36,7 @@ class AdminListingTableRow extends React.Component<Props> {
         </td>
         <td className="admin-table-row--item">
           <BeeLink to={host ? `/admin/users/${host.id}` : ''}>
-            <span>{host ? `${host.firstName} ${host.lastName}` : ''}</span>
+            <span>{host ? `${host.displayName}` : ''}</span>
```

Comment:
This part is actually a change, at least right now; the backend returns `displayName` as `firstName`

We currently hide `lastName` from non-admins at the GraphQL layer. To do something analogous here, we could add some backend logic to compute `displayName` as `${firstName lastName}` for admins and `firstName` for everyone else.

That said, I'm wondering if that information is useful enough here for us to care about that

Reply:
Also, I should note that I haven't tested this change (blocked by another issue, filed ENG-954)

Reply:
Sorry, I think my description was confusing there.

The line of code we are discussing introduces the regression you describe; previously, we showed `firstName` and `lastName` in the admin listings table, with this change we do not.

In the context of the other PR, I realized I could add a [`fullName`](https://github.com/thebeetoken/beenest-backend/pull/711/files#r253246385) property with admin-only permissions to avoid this regression. Think I'll go ahead and do that.

Reply:
works! thanks vic